### PR TITLE
Adding github templates.

### PR DESCRIPTION
**Pull request by weerd**
### What does this PR do?
This PR adds some templates for creating issues and pull-requests in Github. It also adds a contributing file, but we should discuss this further and see what it should contain.

### CODE REVIEW

**Comment by weerd** [comment]
I popped these in a .github folder so it keeps the root directory a bit more organized 💃

File: .github/CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+@TODO: Add contributing guidelines here...
```
Comment (DFurnes) [inline .github/CONTRIBUTING.md]: Do we need this?

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
+### What does this PR do?
+...
+
+
+### Any background context you want to provide?
+...
+
+
+### What are the relevant tickets/cards?
+Fixes #
```
Comment (DFurnes) [inline .github/PULL_REQUEST_TEMPLATE.md]: Should we remove this since we don't generally have issues for work?

Comment (weerd) [inline .github/PULL_REQUEST_TEMPLATE.md]: I added `/cards` to help remind us to associate a trello card to the PR if we're not using a ticket. Maybe just different wording would suffice?

File: .github/CONTRIBUTING.md (hunk 1 above)
Comment (weerd) [inline .github/CONTRIBUTING.md]: 🤷‍♂️

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 2 above)
Comment (DFurnes) [inline .github/PULL_REQUEST_TEMPLATE.md]: Okay, just assuming it's something we'll be deleting more often than not.

Comment (weerd) [inline .github/PULL_REQUEST_TEMPLATE.md]: We should be popping links to the trello cards though. I often forget and this'll be a friendly reminder.

**Pull request merged**